model/command_webhook_test.go

```diff
@@ -44,11 +44,13 @@ func TestCommandWebhookIsValid(t *testing.T) {
 		tmp := h
 		test.Transform()
 		err := h.IsValid()
-		if test.ExpectedError == "" && err != nil {
-			t.Fatal("hook should be valid")
-		} else if test.ExpectedError != "" && test.ExpectedError != err.Id {
-			t.Fatal("expected " + test.ExpectedError + " error")
+
+		if test.ExpectedError == "" {
+			assert.Error(t, err, "hook should be valid")
+		} else {
+			assert.Equal(t, test.ExpectedError, err.Id, "expected "+test.ExpectedError+" error")
```

Comment:
This was an existing issue, but if `err` is `nil`, we'll get a panic here. Can we `require.NotNil` here?

Reply:
Ah true, sorry I missed that. I'll add the `require.NotNil`